authors/apps/articles/tests/test_highlight.py

```diff
@@ -0,0 +1,129 @@
+import json
+from rest_framework.views import status
+from .base import BaseTest
+
+class HighlightTestCase(BaseTest):
+    '''class representing tests for highlighting and commenting on text in an article'''
+    def test_highlight(self):
+        '''test creating highlights and comments'''
+        # test successful highlight and comment
+        response = self.client.post(self.HIGHLIGHT, self.highlight_data, HTTP_AUTHORIZATION=self.non_user_token, format="json")
```

Comment:
Check on pep-8 linting for this line and the whole file. Some lines seem to have exceeded the acceptable limit

Reply:
Review the changes @graycadeau